fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -173,8 +182,57 @@ class EqualizedOdds(ConditionalSelectionRate):
 
     short_name = "EqualizedOdds"
 
+    def __init__(self, ratio=1.0):
+        super(EqualizedOdds, self).__init__()
+        self.ratio = ratio
+
+    def load_data(self, X, y, **kwargs):
+        """Load the specified data into the object."""
+        super().load_data(X, y,
+                          event=pd.Series(y).apply(lambda y: _LABEL + "=" + str(y)),
+                          **kwargs)
+
+
+class ErrorRatio(ConditionalSelectionRate):
```

Comment:
@MiroDudik theoretically this is the ratio of error rates, right? Should it be ErrorRateRatio, and the other one ErrorRateDifference?

Reply:
Agreed. Let's rename `ErrorRatio` -> `ErrorRateRatio`.

Reply:
This is done.